Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN apt-get update -y && \
     apt-get install -y \
         git python3 python3-dev libpython3-dev python3-pip sudo pdsh \
         htop llvm-9-dev tmux zstd software-properties-common build-essential autotools-dev \
-        nfs-common pdsh cmake g++ gcc curl wget tmux less unzip htop iftop iotop ca-certificates ssh \
+        nfs-common pdsh cmake g++ gcc curl wget vim less unzip htop iftop iotop ca-certificates ssh \
```

Comment:
Note: I haven't removed `tmux`, it was just specified twice and added `vim` (which used to be added by default in an older image)